java/configuration/src/main/java/sleeper/configuration/properties/instance/QueryProperty.java

```diff
@@ -80,6 +80,13 @@ public interface QueryProperty {
                     "value given below is 128KiB. This value can be overridden using the query config.")
             .defaultValue("" + (128 * 1024)) // 128 KiB
             .propertyGroup(InstancePropertyGroup.QUERY).build();
+    UserDefinedInstanceProperty DEFAULT_QUERY_WARM_LAMBDA_EXECUTION_PERIOD_IN_MINUTES = Index
+            .propertyBuilder("sleeper.query.warm.lambda.period.minutes")
+            .description("The rate at which the query lambda runs to keep it warm (in minutes, must be >=1).")
```

Comment:
We could mention here that this only applies to the instance when the `KeepLambdaWarmStack` is enabled

Reply:
Wording updated